versions/schematic-2.md

```diff
@@ -0,0 +1,196 @@
+
+##### Fields
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="tileEntityPos"></a>Pos | `integer`[3] | **Required.** The position of the tile entity relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied). Must contain exactly 3 integer values.
+<a name="tileEntityId"></a>Id | `string` | **Required.** The id of the tile entity type defined by this Tile Entity Object, specified as a [Resource Location](#defResourceLocation). This should be used to identify which fields should be required for the definition of this type.
+
+##### Tile Entity Object Example
+
+An example of possible storage of a sign. See the [Minecraft Chunk Format](http://minecraft.gamepedia.com/Chunk_format#Block_entity_format) for a complete listing of data used to store various types of tile entities present in vanilla minecraft. Mods may store additional data or have additional types of tile entities.
+
+```js
+{
+    "ContentVersion": 0,
+    "Pos": [0, 1, 0],
+    "Id": "minecraft:Sign",
+    "Text1": "foo",
+    "Text2": "",
+    "Text3": "bar",
+    "Text4": ""
+}
+```
+
+#### <a name="entityObject"></a>Entity Object
+An object to specify an instance of an [entity type](#defEntityType) within the region. Entities usually occupy a three dimensional space within the region and have many characteristics, varied by the entity type, however the structure will be the same as used by the [Minecraft Chunk Format](https://minecraft.gamepedia.com/Chunk_format#Entity_format), dependent on the `Data Version` the schematic is created from.
+
+##### Fields
+Field Pattern | Type | Description
+---|:---:|---
+<a name="entityPos"></a>Pos | `double`[3] | **Required.** The position of the entity relative to the [0, 0, 0] position of the schematic (without the offset applied). Must contain exactly 3 double values.
+<a name="entityId"></a>Id | `string` | **Required.** The id of the entity type defined by this Entity Object, specified as a [Resource Location](#defResouceLocation). This should be used to identify which fields should be required for the definition of this type.
+<a name="entityExtra"></a>Extra | `unknown` | **Optional** The extra information related to an entity that otherwise is either defaulted or required by the entity associated with the [entity type](#defEntityType)
+
+##### Entity Object Example
+
+An example of possible storage of a creeper. See the [Minecraft Chunk Format](https://minecraft.gamepedia.com/Chunk_format#Entity_format for a complete listing of data used to store various types of entities present in vanilla Minecraft. Mods may store additional data or have additional types of entities.
+
+```json
+{
+    "Pos": [15.0043293D, 68.000321D, 40.452D],
+    "Id": "minecraft:creeper",
+    "Motion": [0.00203, 0.00203, 1.000],
```

Comment:
maybe worth clarifying that these are doubles since the `D` notation was used for Pos? on the other hand `Motion` is an "implementation specific"/extra tag and it could technically be anything.